packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -252,6 +252,47 @@ Future<XcodeBuildResult> buildXcodeProject({
     );
   }
 
+  // If buildNumber is not specified, keep the project untouched.
+  if (buildInfo.buildNumber != null) {
+    final Status buildNumberStatus =
+    logger.startProgress('Running agvtool new-version...', expectSlowOperation: true);
```

Comment:
I wouldn't use the words new-version and new-marketing-version. It's only used in the agvtool. I would stick to common words like CFBundleVersion and CFBundleShortVersionString